iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/helper/DummyAndroidRunner.java

```diff
@@ -44,30 +29,8 @@ public void labelSnapshot()
     @TestGroup15
     @TestGroup16
     @TestGroup17
```

Comment:
q - we need fewer test groups now because...? can we parallelize more stuff in the android emulator, or do we spin up more emulators?

Reply:
We used to have separate test groups for device and module tests. For instance, we had a SendMessagesDeviceAndroidRunner and a SendMessagesModuleAndroidRunner, but now those two are combined into just SendMessagesAndroidRunner. On top of that, several test groups contained only a few tests, so they were also grouped together instead of given their own test group

Reply:
RegistryManager tests and Service Client tests, for instance, don't need their own test group anymore. They can share